Moller-Plesset/README.md

```diff
@@ -0,0 +1,22 @@
+Moller-Plesset Perturbation Theory
+=================
+Moller-Plesset perturbation theory is a post-SCF quantum chemistry method which
+adds dynamic electron correlation to improve upon SCF level wavefunctions. This
+method is a workhose of quantum chemistry as it has proved to be remarkably
+accurate considering its overall cost and is the basis for many theories built
+ontop of this result.
+
+### Included Reference Implementations
+ - `MP2.py`: Second-order Moller-Plesset perturbation theory
+ - `DF-MP2.py`: MP2 utilizing density-fitting to reduce its overall cost.
+ - `MP3.py`: Third-order Moller-Plesset perturbation theory
+ - `MP3_SO.py`: MP3 in spin-orbital formalism that simplifies the equations and provides contrast to the spin-summed version above.
+ - `MPn.py`: An example on how to automate the contraction of higher order MP theory.
```

Comment:
Convention I like is that hyphens separate words within the same "field", while underscores separate "fields".  Since the SO just expands on which kind of MP3 it is, it's in the same "field" and separated by a hyphen: `MP3-SO.py`.  For the tutorials, though, the chapter number is a different "field" than the tutorial name: `5a_conventional-mp2.ipynb`.  Is that too complicated?